packages/opentelemetry-metrics/src/export/ConsoleMetricExporter.ts

```diff
@@ -15,13 +15,32 @@
  */
 
 import { MetricExporter, MetricRecord, Distribution, Histogram } from './types';
+import * as api from '@opentelemetry/api';
 import { ExportResult } from '@opentelemetry/core';
+import { PushControllerConfig, PushController } from './controllers';
+import { MeterProvider } from '..';
+import { MeterConfig } from '../types';
 
 /**
  * This is implementation of {@link MetricExporter} that prints metrics data to
  * the console. This class can be used for diagnostic purposes.
  */
 export class ConsoleMetricExporter implements MetricExporter {
+  /**
+   * Install console export pipeline to global metrics api.
+   * @param config Default meter configuration
+   */
+  static installExportPipeline(
+    config?: PushControllerConfig,
+    meterConfig?: MeterConfig
+  ) {
```

Comment:
missing return type - I think it should be the same interface that you have in prometheus, we might use it later for CollectorMetricExporter as well